adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -816,18 +816,7 @@ private IWindowComponent wrapActivity(final Activity activity) {
             throw new IllegalArgumentException("activity");
         }
 
-        return new IWindowComponent() {
-            private Activity mRefActivity = activity;
```

Comment:
Thank you for your contribution. I think we would achieve the same result by changing mRefActivity to be a WeakReference. do you agree?